demo/search_ranges.R

```diff
@@ -7,6 +7,7 @@ library(RXapian)
 db<-c("path/to/db")
 
 vrp1<-list(type="proc.num", value=c(0), check.str="mm", flags=c("RP_SUFFIX"))
+#If a Xapian version earlier than 1.3.6 is used replace vrp1 with: vrp1<-list(type="proc.num", value=c(0), check.str="mm", prefix=FALSE)
```

Comment:
I didn't get you?

On Tue, Aug 9, 2016 at 3:22 PM, James Aylett <notifications@github.com>
wrote:

> In demo/search_ranges.R
> <https://github.com/amandaJayanetti/RXapian/pull/25#discussion_r74029463>:
>
> > @@ -7,6 +7,7 @@ library(RXapian)
> >  db<-c("path/to/db")
> >
> >  vrp1<-list(type="proc.num", value=c(0), check.str="mm", flags=c("RP_SUFFIX"))
> > +#If a Xapian version earlier than 1.3.6 is used replace vrp1 with: vrp1<-list(type="proc.num", value=c(0), check.str="mm", prefix=FALSE)
>
> For pre-1.3.6, you could explicitly check flags=c("RP_SUFFIX") yourself,
> to provide the same API to R users?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amandaJayanetti/RXapian/pull/25/files/8b4adf893b924b6f2db7b7766841af1266c7734f..4c44e39595bb06b87e1d5a33c427c21eef6ab74d#r74029463>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH_jNkoJ3wJ9rJir6MaQsOv8L0nbANEtks5qeE3NgaJpZM4Jf4Xo>
> .
>